dev/com.ibm.ws.jdbc_fat_postgresql/fat/src/com/ibm/ws/jdbc/fat/postgresql/FATSuite.java

```diff
@@ -78,10 +53,31 @@ private static boolean tryDockerHost(ExternalTestService dockerService) {
         String m = "tryDockerHost";
         String dockerHostURL = "tcp://" + dockerService.getAddress() + ":" + dockerService.getPort();
         System.setProperty("DOCKER_HOST", dockerHostURL);
+        try {
+            File certDir = new File("docker-certificates");
```

Comment:
Im guessing this directory isnt included in the FAT Output.

Reply:
it is, but there are already secrets like this included in FAT output